CommonMC/src/CosmicTimeOffset_module.cc

```diff
@@ -82,6 +93,24 @@ namespace mu2e {
     std::unique_ptr<SimTimeOffset> toff(new SimTimeOffset(flatTime_.fire()));
     if( verbosityLevel_ > 1) std::cout << "CosmicTimeOffset " << toff->timeOffset_ << std::endl;
     event.put(std::move(toff));
+
+    if (addTimeOffset_ ) {
+      // Add offset to all particles
+      art::Handle<mu2e::GenParticleCollection> cosmicParticles;
+      event.getByLabel(cosmicModuleLabel_, cosmicParticles);
+      const GenParticleCollection &particles(*cosmicParticles);
+      std::unique_ptr<GenParticleCollection> offsetParticles(new GenParticleCollection);
+      const float timeOffset = flatTime_.fire();
```

Comment:
Actually this needs to be the same offset value is stored in the SimTimeOffset, for consistency.

Reply:
Done